ui/Gruntfile.js

```diff
@@ -48,7 +48,10 @@ module.exports = function(grunt) {
         tasks: ['wiredep']
       },
       js: {
-        files: ['<%= yeoman.app %>/scripts/{,*/}*.js'],
+        files: [
+          '<%= yeoman.app %>/scripts/{,*/}*.js',
+          '<%= yeoman.app %>/scripts/components/{,*/}*.js'
```

Comment:
is the first match pattern not picking up on components too? I would have thought the `{,*/}` would traverse sub-directories